nautobot/core/utils/requests.py

```diff
@@ -171,6 +183,8 @@ def normalize_querydict(querydict, form_class=None):
             ):
                 # Even though there's only a single value in the querydict for this key, the form wants it as a list
                 result[key] = value_list
+            elif filterset is not None and filterset.filters.get(key) and not is_single_choice_field(filterset, key):
```

Comment:
Would be good to enhance `nautobot.core.tests.test_utils` `test_normailze_querydict()` to cover this case.